src/NoteParser.ts

```diff
@@ -57,11 +62,7 @@ export class Note {
   fsPath: string;
   data: string | undefined;
   refCandidates: Array<RefCandidate> = [];
-  title: {
```

Comment:
This was suddenly error-ing, some sort of linter issue. I resolved by declaring a new type.

Also, this file got a lot of linter diff lines.